public/app/features/admin/AdminOrgsTable.tsx

```diff
@@ -28,7 +28,13 @@ export const AdminOrgsTable: FC<Props> = ({ orgs, onDelete }) => {
               <a href={`admin/orgs/edit/${org.id}`}>{org.name}</a>
             </td>
             <td className="text-right">
-              <Button variant="destructive" size="sm" icon="times" onClick={() => setDeleteOrg(org)} />
+              <Button
+                variant="destructive"
+                size="sm"
+                icon="times"
+                onClick={() => setDeleteOrg(org)}
+                aria-label="Delete org"
```

Comment:
We could use `aria-label="Delete organization"` here instead

Reply:
I was of two minds about this but decided to go with "org" since it seems to be what we consistently refer to it as